examples/elemental/build/kubernetes.yaml

```diff
@@ -2,14 +2,11 @@ manifests:
   - https://raw.githubusercontent.com/rancher/local-path-provisioner/v0.0.31/deploy/local-path-storage.yaml
 helm:
   charts:
-    - name: cert-manager
-      chart: cert-manager
-      version: v1.17.2
-      namespace: cert-manager
-      repository: cert-manager-repo
-      values: |-
-        crds:
-          enabled: true
+    - name: nginx
+      version: 20.1.3
```

Comment:
potentially quoted as a string here?

Reply:
We could do it but it's not really an issue.